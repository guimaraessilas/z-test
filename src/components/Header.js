import React from "react";
import { View, Text, StyleSheet } from "react-native";

const Header = (props) => {
  return (
    <View style={styles.header}>
      <Text style={styles.headerText}>{props.title}</Text>
    </View>
  );
};

export default Header;

const styles = StyleSheet.create({
  header: {
    width: "100%",
    height: 60,
    backgroundColor: "#0088cc",
    alignItems: "center",
    justifyContent: "center",
  },
  headerText: {
    fontSize: 26,
    fontWeight: "700",
    color: "white",
  },
});
