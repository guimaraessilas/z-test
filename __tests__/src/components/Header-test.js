import React from "react";
import Header from "../../../src/components/Header";
import renderer from "react-test-renderer";

describe("Should test Header Component ", () => {
  describe("snapshot", () => {
    it("render correctly", () => {
      const tree = renderer.create(<Header />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
