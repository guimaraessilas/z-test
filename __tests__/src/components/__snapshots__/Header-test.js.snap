// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test Header Component  snapshot render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0088cc",
      "height": 60,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 26,
        "fontWeight": "700",
      }
    }
  >
    Zeca Chaceiro
  </Text>
</View>
`;
